.gitattributes

```diff
@@ -4,3 +4,5 @@ stglib/_version.py export-subst
 stglib/tests/data/055109_20220808_1605.zip filter=lfs diff=lfs merge=lfs -text
 stglib/tests/data/055109_20220808_1605_config.yaml filter=lfs diff=lfs merge=lfs -text
 stglib/tests/data/gatts_055109_20220808_1605.txt filter=lfs diff=lfs merge=lfs -text
+stglib/tests/data/*.cdf filter=lfs diff=lfs merge=lfs -text
```

Comment:
Ideally the test files should be listed in the .gitattributes file in stglib/tests/data, not this .gitattributes file.

Reply:
moved these lines to .gitattributes file in stglib/tests/data folder